installing/installing_aws/installing-aws-outposts.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-aws-outposts"]
+= Installing a cluster on AWS with compute nodes on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts
+
+toc::[]
+
+In {product-title} version 4.14, you could install a cluster on
+Amazon Web Services (AWS) with compute nodes running in AWS Outposts as a Technology Preview.
+As of {product-title} version {product-version}, this installation method is no longer supported.
```

Comment:
Should the `product-version}` variable be replaced with the first version that no longer supports this feature? Technically the phrase "As of" implies we're talking about the first version that dropped support, not whether it's supported in the current version a user is looking at docs for.

```suggestion
As of {product-title} version 4.15, this installation method is no longer supported.
```